launchd/keybase.keybot.promote.darwin.plist

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>keybase.keybot.promote.darwin</string>
```

Comment:
I was following the convention of keybase.keybot.[thing].[platform].plist. like `keybase.keybot.build.darwin`. It's possible to have keybot sending a promote command for other platforms too.